descartes_light/core/include/descartes_light/solvers/ladder_graph/impl/ladder_graph_solver.hpp

```diff
@@ -140,7 +140,11 @@ BuildStatus LadderGraphSolver<FloatType>::buildImpl(
       std::stringstream s;
       s << vert_idx << ": " << *pos;
 
-      CONSOLE_BRIDGE_logDebug("%s", s.str().c_str());
+      if (console_bridge::getLogLevel() == console_bridge::LogLevel::CONSOLE_BRIDGE_LOG_DEBUG)
+      {
+        std::cout << s.str().c_str() << std::endl;
+      }
+
```

Comment:
Why is this change necessary?

Reply:
The default console logging (maybe this is just something because of roslaunch) cuts off the printout after so many characters. I couldn't even get it to display all the information about the first observed collision. I had to do the same thing when I made changes to the tesseract trajectory collision checker.

Reply:
For reference, `console_bridge` appears to have a [1024 character limit when logging](https://github.com/ros/console_bridge/blob/7048ab4539736416fa423fd0a30744d4902333a9/src/console.cpp#L116C14-L116C35).

I would prefer not to log directly to `cout` so users can specify how to route the console bridge output for their own application (e.g., to terminal by default, or to GUI loggers, etc.). Per our discussion, we want to revise the `tesseract` sampler implementation that is exceeding this character limit, so let's address that first and come back to this issue afterwards if it remains a problem